<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'security.authentication.success_handler.api.json_login' shared service.

include_once $this->targetDirs[3].'/vendor/symfony/security-http/Authentication/AuthenticationSuccessHandlerInterface.php';
include_once $this->targetDirs[3].'/vendor/symfony/security-http/Authentication/CustomAuthenticationSuccessHandler.php';

return $this->privates['security.authentication.success_handler.api.json_login'] = new \Symfony\Component\Security\Http\Authentication\CustomAuthenticationSuccessHandler(($this->privates['lexik_jwt_authentication.handler.authentication_success'] ?? $this->load('getLexikJwtAuthentication_Handler_AuthenticationSuccessService.php')), [], 'api');
